tests/test_unstupify_path.py

```diff
@@ -10,6 +10,7 @@
 
 HERE = os.path.realpath(__file__)
 DIR, FILE = os.path.split(HERE)
+os.chdir(DIR)  # rex assumes that all tests are run from /tests/
```

Comment:
Was this necessary because some other tests were changing directories and causing this one to subsequently break? If so, we should consider fixing the problem in the offending tests. I am worried that if we ever add a file `test_u[a-m]*.py`, it will break b/c this fix hasn't been applied yet.

Reply:
No, it's that just this test file in particular test relative path manipulation and therefore assume that the cwd in in /tests/. I'm not too worried about it, any other real test should be cwd independent, and our test suite will catch it if not.

Reply:
I won't object, but some of the most difficult-to-traceback and non-intuitive errors I have encountered in recent history had to do with cwd manipulation during test runtime. Maybe just a mental note in case some weird error ever pops up.